waku/v2/node/config.nim

```diff
@@ -8,9 +8,15 @@ import
   libp2p/crypto/secp,
   nimcrypto/utils,
   eth/keys,
-  ../protocol/waku_rln_relay/waku_rln_relay_types,
   ../protocol/waku_message
 
+## TODO: properly address these import once zerokit rln is merged
+when defined(rln) or (not defined(rln) and not defined(rlnzerokit)):
+  import ../protocol/waku_rln_relay/waku_rln_relay_types
+
+when defined(rlnzerokit):
+  import ../protocol/waku_rln_relay/rlnzerokit/waku_rln_relay_types
+
```

Comment:
```suggestion
## TODO: properly address these import once zerokit rln is merged
when defined(rln):
  import ../protocol/waku_rln_relay/waku_rln_relay_types
else:
  when defined(rlnzerokit):
     import ../protocol/waku_rln_relay/rlnzerokit/waku_rln_relay_types

```

Reply:
Such separation is not needed when the same file is used.

Reply:
Once we remove duplicate waku-rln-relay-types.nim files, then there is no need for conditional import.